src/components/paragraph/sul-home-banner/sul-home-banner.client.tsx

```diff
@@ -27,9 +27,23 @@ export const SulHomeBannerRandomClient = ({children, ...props}: Props) => {
 export const SulHomeBannerFormClient = () => {
   const [formAction, setFormAction] = useState("/all")
   const inputId = useId()
+
+  // Google Analytics: Track the selected search option when the user submits the form.
+  // Helps measure actual search behavior by logging which resource option was used.
+  const handleSubmit = () => {
+    if (typeof window !== "undefined" && window.gtag) {
+      window.gtag("event", "search_option_selected", {
```

Comment:
Should use the provided functions. https://nextjs.org/docs/app/guides/third-party-libraries#sending-events-1